src/R/Sql/Impl/Publish/DacPackageServices.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.ComponentModel.Composition;
+using Microsoft.R.Components.Sql.Publish;
+
+namespace Microsoft.VisualStudio.R.Sql.Publish {
+    [Export(typeof(IDacPackageServices))]
```

Comment:
If you want to do it through MEF, the right way is to load "Microsoft.VisualStudio.R.Sql.dll" into its own assembly catalog that doesn't pollute main VS MEF catalog with information about incorrectly loaded assembly. With only one type required, I'd prefer simple reflection.